lib/cli.js

```diff
@@ -13,7 +13,6 @@ const chalk = require('chalk');
 const cosmiconfig = require('cosmiconfig');
 const debug = require('debug')('webpack-serve');
 const meow = require('meow');
-const merge = require('merge-options').bind({ concatArrays: true });
```

Comment:
Happy to see it go, but I think this is still needed because `Object.assign` just shallow copies

Reply:
but it was used nowhere then, so I just removed it

Reply:
You're totally write 🙃. Could you check if `merge-options` can be removed and remove from the package dependencies aswell please ?

Reply:
we are still using it in `options.js` https://github.com/webpack-contrib/webpack-serve/blob/master/lib/options.js#L4